bioio_base/standard_metadata.py

```diff
@@ -103,3 +109,118 @@ def to_dict(self) -> dict:
             self.FIELD_LABELS[field]: getattr(self, field)
             for field in self.FIELD_LABELS
         }
+
+
+def binning(ome: OME) -> Optional[str]:
+    """
+    Extracts the binning setting from the OME metadata.
+
+    Returns
+    -------
+    Optional[str]
+        The binning setting as a string. Returns None if not found.
+    """
+    try:
+        # DetectorSettings under each Channel holds the binning info
+        channels = ome.images[0].pixels.channels or []
+        for channel in channels:
+            ds = channel.detector_settings
+            if ds and ds.binning:
+                return str(ds.binning.value)
+    except Exception as exc:
+        log.warning("Failed to extract Binning setting: %s", exc, exc_info=True)
+    return None
+
+
+def imaged_by(ome: OME) -> Optional[str]:
+    """
+    Extracts the name of the experimenter (user who imaged the sample).
+
+    Returns
+    -------
+    Optional[str]
+        The username of the experimenter. Returns None if not found.
+    """
+    try:
+        img = ome.images[0]
+        # Prefer explicit ExperimenterRef if present
+        if img.experimenter_ref and ome.experimenters:
+            exp = next(
+                (e for e in ome.experimenters if e.id == img.experimenter_ref.id), None
+            )
+            if exp and exp.user_name:
+                return exp.user_name
+        # Fallback to first Experimenter
+        if ome.experimenters:
+            return ome.experimenters[0].user_name
+    except Exception as exc:
+        log.warning("Failed to extract Imaged By: %s", exc, exc_info=True)
+    return None
+
+
+def imaging_date(ome: OME) -> Optional[str]:
+    """
+    Extracts the acquisition date from the OME metadata.
+
+    Returns
+    -------
+    Optional[str]
+        The acquisition date in ISO format (YYYY-MM-DD) adjusted to the local
+        system timezone.
+
+        None: if the acquisition date is not found or cannot be parsed.
+    """
+    try:
+        img = ome.images[0]
```

Comment:
Is `images` necessarily sorted? If not, perhaps we should choose the first acquisition chronologically?

Reply:
I have yet to see a file that it is not sorted for.